ui/components/multichain/edit-networks-modal/edit-networks-modal.js

```diff
@@ -0,0 +1,84 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { TextVariant } from '../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { getNonTestNetworks, getTestNetworks } from '../../../selectors';
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  Checkbox,
+  Text,
+  Box,
+} from '../../component-library';
+import { NetworkListItem } from '..';
+
+export const EditNetworksModal = ({ onClose }) => {
```

Comment:
i suggest to add props type here

Reply:
It uses NetworkListItem components internally which is not TS compatible. So, it's in JS for now